src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -28,11 +28,19 @@ public static IConfigurationBuilder AddUserSecrets<T>(this IConfigurationBuilder
             => configuration.AddUserSecrets(typeof(T).GetTypeInfo().Assembly);
 
         /// <summary>
+        /// <para>
+        /// This method is obsolete and will be removed in a future version.
+        /// The recommended alternative is Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets&lt;T&gt;()
```

Comment:
This can be actual `cref` tags, no? That way don't need to type in the full type name.